<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgConvexDecomposition</class>
 <widget class="QDialog" name="CQDlgConvexDecomposition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convex Decomposition</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqExtraDistPoints">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Add extra points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. nb of clusters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqClusters">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. concavity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqConcavity">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqExtraFacesPoints">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Add extra face points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. connection distance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqConnectDist">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqTargetBlabla">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Target nb of triangles of decimated mesh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. nb of vertices / convex hull</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqMaxHullVertices">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Small cluster detection threshold (%)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqSmallClusterThreshold">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqIndividualComponents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Handle grouped shape components individually</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqRandomColors">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Apply random colors for easier entity distinction</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. iterations</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqMaxIterations">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqIndividualComponents</tabstop>
  <tabstop>qqRandomColors</tabstop>
  <tabstop>qqExtraDistPoints</tabstop>
  <tabstop>qqExtraFacesPoints</tabstop>
  <tabstop>qqClusters</tabstop>
  <tabstop>qqConcavity</tabstop>
  <tabstop>qqConnectDist</tabstop>
  <tabstop>qqTargetBlabla</tabstop>
  <tabstop>qqMaxHullVertices</tabstop>
  <tabstop>qqSmallClusterThreshold</tabstop>
  <tabstop>qqMaxIterations</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
